readthedocs/projects/views/private.py

```diff
@@ -468,18 +469,28 @@ def _is_last_user(self):
         return self.get_queryset().count() <= 1
 
 
-class ProjectUsersCreateList(ProjectUsersMixin, FormView):
+class ProjectUsersCreateList(SuccessMessageMixin, ProjectUsersMixin, FormView):
 
     template_name = 'projects/project_users.html'
+    success_message = _("Invitation sent")
 
     def form_valid(self, form):
         form.save()
-        return HttpResponseRedirect(self.get_success_url())
+        return super().form_valid(form)
```

Comment:
In this case, I think it's not needed to overload this method. The default should do what we want.

Reply:
This is a form view, not a model view, so we need to call save manually.

Reply:
👍🏼 In that case, would be good to add a comment here explaining that.

Reply:
Comment added :)